typings.d.ts

```diff
@@ -0,0 +1,9 @@
+declare module 'react-media' {
+  interface MediaProps {
+    query: string;
+    defaultMatches?: boolean;
+    children: ((matches: boolean) => JSX.Element) | JSX.Element
```

Comment:
`children` should be optional, and i believe these should specify React.ReactNode instead of JSX.Element